### PR TITLE
PHPUnit 7

### PR DESCRIPTION
**Pull request by martijngastkemper**
PHPUnit 8 isn't possible because Garp Functional must be PHP 7.1 compatible.

### CODE REVIEW

File: tests/PropTest.php [hunk 1]
```diff
@@ -107,7 +107,7 @@ public function offsetUnset($offset) {
             array(null, 'foo', $obj),
             array('12345', 'bar', $obj),
             array(2012, 'year', $carbonInstance),
-            array(248, 'dayOfYear', $carbonInstance),
+            array(249, 'dayOfYear', $carbonInstance),
```
Comment (harmenjanssen) [inline tests/PropTest.php:110]: How is this possible? Did Carbon get an update that made its day-of-year index one-based instead of zero-based?

Comment (martijngastkemper) [reply tests/PropTest.php:110]: Yes. The second from below: https://carbon.nesbot.com/docs/#api-carbon-2

Comment (harmenjanssen) [reply tests/PropTest.php:110]: I guess it makes sense when considering "day of the year".

**Review by harmenjanssen — APPROVED** [review]
LGTM!

**Pull request merged**